package.json

```diff
@@ -56,7 +56,8 @@
     "build": "turbo run build",
     "vercel-build": "mkdir -p public && echo '<a href=\"https://vercel.com/import\">Import</a>' > public/output.html",
     "pre-commit": "lint-staged",
-    "test-unit": "node utils/run.js test-unit",
+    "test": "jest",
+    "test-unit": "jest && node utils/run.js test-unit",
```

Comment:
Is running `jest` going to pickup too many tests? Seems like we should only run unit tests instead of all here